hack/test-templates.sh

```diff
@@ -389,6 +407,10 @@ if [[ -n ${CHECKS["snapshot-offline"]} ]]; then
 	limactl start "$NAME"
 fi
 
+if [[ -n ${CHECKS["vz-selinux"]} ]]; then
```

Comment:
This should check `limactl shell "$NAME" getenforce`

Reply:
We need to run this only for vz based VMs and so we filter the check based on the vm name.

Reply:
Can we just check `limactl ls --json "$NAME" | jq -r .vmType` ?